client/src/components/RatingsAndReviews/AddReview.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+import styled from 'styled-components';
+
+const StyledInput = styled.input`
+font-family: 'Helvetica', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif, Helvetica, sans-serif;
```

Comment:
Later you can refactor to create a global style for the font family. Just a suggestion :)

Reply:
Yeah planning to do so next